spark-k8s/versions.py

```diff
@@ -35,4 +35,22 @@
         "tini": "0.19.0",
         "hbase_connector": "1.0.1",
     },
+    {
+        "product": "4.0.0",
+        "java-base": "17",
+        "java-devel": "17",
+        "python": "3.11",
+        "hadoop/hadoop": "3.4.1",
+        "hbase": "2.6.2",
+        "aws_java_sdk_bundle": "2.24.6",
+        "azure_storage": "7.0.1",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-azure/3.3.4
+        "azure_keyvault_core": "1.0.0",  # https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage/7.0.1
+        "jackson_dataformat_xml": "2.15.2",  # https://mvnrepository.com/artifact/org.apache.spark/spark-core_2.13/3.5.1
+        "stax2_api": "4.2.1",  # https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.15.2
+        "woodstox_core": "6.5.1",  # https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.15.2
+        "vector": "0.47.0",
+        "jmx_exporter": "1.3.0",
+        "tini": "0.19.0",
+        "hbase_connector": "1.0.1",
```

Comment:
If we go with my previous suggestion we would probably update it to
```suggestion
        # hbase_connector is not yet supported in Spark 4, see https://github.com/apache/hbase-connectors/pull/130 for updates
```